geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py

```diff
@@ -75,6 +75,8 @@ def main():
     if options.nondocker:
         print("- nondocker")
     print("- git_remote=" + options.git_remote)
+    if options.use_makefile:
+        print("- use_makefile")
     print("- makefile=" + options.makefile)
```

Comment:
looks a little bit strange, can you verify ?

What's the difference between "makefile" and "use_makefile" ? (also for the argument parser below (line 112))

Reply:
The makefile is the make file used by c2cupgrade to lunch the build tasks, use_makefile is true if we lunch c2cupgrade true the makefile, it's used only tho display the commends used to run the different steps.

Reply:
Ok, thanks

Reply:
any idea?

Reply:
Not really :sweat_smile: